src/overmapbuffer.h

```diff
@@ -53,6 +55,20 @@ struct city_reference {
     int get_distance_from_bounds() const;
 };
 
+struct camp_reference {
+    static const camp_reference invalid;
+    /** The camp itself, points into @ref overmap::cities */
+    const struct om_camp *camp;
```

Comment:
why the pointer to a structure containing the basecamp* ?  Why not just the basecamp * and done?